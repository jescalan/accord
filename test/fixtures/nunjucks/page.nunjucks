{% extends "_parent.html" %}
{% block header %}Accord{% endblock %}
