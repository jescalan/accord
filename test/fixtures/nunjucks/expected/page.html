<header>
  Accord
</header>

<footer>
  Footer
</footer>
